.travis.yml

```diff
@@ -10,16 +9,37 @@ php:
   - hhvm
  
 matrix:
-    allow_failures:
-        - php: hhvm
+  fast_finish: true
+  include:
+    - php: 5.3
+      env: SYMFONY_VERSION='2.3.*' FOSUSERBUNDLE_VERSION='~1.3'
```

Comment:
you don't actually need testing all FOSUserBundle versions on all Symfony versions IMO. the compatibility between FOSUB and Symfony is not something you are responsible for testing.